src/features/CodeLinkFeature.ts

```diff
@@ -24,10 +28,40 @@ const toggleCodeLinkActivationCommandName =
 	"hediet.vscode-drawio.toggleCodeLinkActivation";
 const linkCodeWithSelectedNodeCommandName =
 	"hediet.vscode-drawio.linkCodeWithSelectedNode";
-
+const linkSymbolWithSelectedNodeCommandName =
+	"hediet.vscode-drawio.linkSymbolWithSelectedNode";
 const linkFileWithSelectedNodeCommandName =
 	"hediet.vscode-drawio.linkFileWithSelectedNode";
 
+const symbolList = [
+	'symbol-file',
+	'symbol-module',
+	'symbol-namespace',
+	'symbol-package',
+	'symbol-class',
+	'symbol-method',
```

Comment:
Better documentation please or better name (e.g. `symbolNameMap`) ;)
Maybe a record `Record<Symbol, string>` is better here to indicate the purpose.